app/pages/500.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import StaticLayout from "components/Layout/StaticLayout";
+import getConfig from "next/config";
+import Link from "next/link";
+import { Button } from "@button-inc/bcgov-theme";
+import { useRouter } from "next/router";
+import Alert from "@mui/material/Alert";
+
+const Custom500 = () => {
+  const supportEmail = getConfig()?.publicRuntimeConfig.SUPPORT_EMAIL;
+  const mailtoLink = `mailto:${supportEmail}?subject=Reporting an error`;
+
+  const router = useRouter();
+
+  return (
+    <>
+      <StaticLayout>
+        <Alert severity="error" icon={false}>
+          <h3>Sorry, something went wrong</h3>
```

Comment:
```suggestion
          <h2>Sorry, something went wrong</h2>
```
Jumping heading levels is an accessibility issue, so this should be an `h2`